lib/Parse/SyntaxParsingCache.cpp

```diff
@@ -15,6 +15,45 @@
 using namespace swift;
 using namespace swift::syntax;
 
+ArrayRef<TriviaPiece> getNextChildsLeadingTrivia(const Syntax *Node) {
```

Comment:
Would be nice to make this `getNextChild`, because that's a traversal I've found very necessary.

Reply:
Sounds good. I will keep it in mind when I refactor these methods to provide a (hopefully faster) cache based lookup.